cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/RatioTapChangerConversion.java

```diff
@@ -170,8 +171,8 @@ private double fixing(PropertyBag point, String attr, double defaultValue, Strin
         double value = point.asDouble(attr, defaultValue);
         if (Double.isNaN(value)) {
             fixed(
-                    "RatioTapChangerTablePoint " + attr + " for step " + step + " in table " + tableId,
-                    "invalid value " + point.get(attr));
+                "RatioTapChangerTablePoint " + attr + " for step " + step + " in table " + tableId,
+                () -> "invalid value " + point.get(attr));
```

Comment:
Same remark here

Reply:
@lcosson: I think there is no reason to use a supplier here.

Reply:
My bad, correcting this...